web/rootfs/defaults/meet.conf

```diff
@@ -98,7 +98,20 @@ location ^~ /etherpad/ {
     proxy_set_header Connection "upgrade";
     proxy_set_header X-Forwarded-For $remote_addr;
 
-    proxy_pass {{ .Env.ETHERPAD_URL_BASE }}/;
+    proxy_pass http://etherpad.meet.jitsi:9001/;
+}
+
+# WBO
+location ^~ /wbo/ {
+    proxy_buffering off;
+    proxy_cache_bypass $http_upgrade;
+
+    proxy_http_version 1.1;
+    proxy_set_header Upgrade $http_upgrade;
+    proxy_set_header Connection "upgrade";
+    proxy_set_header X-Forwarded-For $remote_addr;
+
+    proxy_pass http://wbo.meet.jitsi/;
```

Comment:
Why is the port not necessary here?

Reply:
Because the WBO container esposes port 80, and it doesn't need specifying since it's the default for HTTP.